### ISSUE TITLE
Searching in Betterbird 128 is slower(!) than Thunderbird 132

### ISSUE THREAD

**Issue by pfolk** [Betterbird/thunderbird-patches#358]
I was trying out the Thunderbird 128 and found that Quick Filter functionality was much improved.  It is now faster than Betterbird (release and beta versions).  My 191k-message inbox takes 10s to search in Betterbird beta (based on TB128), but only 5s in Thunderbird 128.3.3 (which, per https://bugzilla.mozilla.org/show_bug.cgi?id=1890230, is now maybe-safe).  I'm guessing that BB has its own more-advanced search code that needs to be redone to benefit from changes from TB115 to TB128, but I don't have any actual reason for that suspicion.

**Comment by Betterbird** [comment]
Yes, quick filter search is a little slower in BB as BB shows the results as the are found, and not "in bulk" at the end as TB does. That improves the feedback. Note that TB 128 has zero QF feedback.

If it gets worse than a 5 second difference, please let us know.

**Comment by pfolk** [comment]
For the recrod TB128 shows a first set of results after ~2s, same as Betterbird as far as I can tell, so they are both incremental.  Difference is BB then fills in the rest over 8s and TB over 3s.  The absolute numbers aren't terribly different and I do have a LOT of messages =) but 2.5x the search time seems to point to an algorithmic difference not just a few incremental updates.  Anyway, thanks for the quick response!

**Comment by Betterbird** [comment]
There are no algorithmic differences, the only difference is in the incremental/in bulk feedback. TB is not incremental at all, but it seems that a first chunk is displayed after which there is no feedback any more.

We have a folder with 300.00 message in which we do a body search. BB shows the search results as they are found, TB appears to show a first chunk, then there is no feedback for ages.

Please try a few combinations and watch out for the feedback you're getting. BB's focus is on providing feedback, not the fastest result. There are searches in TB which take minutes during which the user received zero feedback.
